source/_data/image_clients.yml

```diff
@@ -1,4 +1,18 @@
 # Use Markdown in the blurb if necessary for links, etc. Each list entry in the blurb gets a <p>.
+- name: Mirador
+  uri: https://github.com/ProjectMirador/mirador
+  blurb:
+  - >
+    Mirador is an open source image, Javascript and HTML5 viewer that delivers
+    high resolution images in a workspace that enables comparison of multiple
+    images from multiple repositories.  Mirador is fully IIIF-compatible. Visit
+    the [Mirador demo](http://showcase.iiif.io/viewer/mirador/) to see it in
+    action.
+    - name: Universal Viewer
+  uri: https://github.com/universalviewer
```

Comment:
For some reason this change means that when you click on the Mirador link on the preview page: https://preview.iiif.io/root/apps-demos-update/apps-demos/#image-viewing-clients you get taken to the UV page... (First link on the apps-demos page)